src/main/java/com/itschool/eventmanagment/exceptions/ParticipantNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package com.itschool.eventmanagment.exceptions;
+
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+
+@ResponseStatus(value = NOT_FOUND, reason = "Participant not found")
```

Comment:
If we have the message here, this means there is no reason having a String parameter in the constructor.